docs/protocols/java_object_graph_spec.md

```diff
@@ -45,16 +56,24 @@ field of a class, only `NULL FLAG` and ` NOT_NULL VALUE FLAG` will be used.
 
```

Comment:
The REF FLAG description should indicate whether the 'unsigned ref id' is an unsigned variable integer or an fixed-length integer.

Reply:
updated